services/sleeper/src/sleeper/main.py

```diff
@@ -0,0 +1,110 @@
+import os
+import json
+import random
+import time
+import subprocess
+
+
+def get_from_environ(key, default=None):
+    """Returns a value from the environ if not found None"""
+    return os.environ.get(key, default)
+
+
+def get_random_sleep():
+    """Returns a random amount of sleep between 1 and 9"""
+    return random.randint(1, 9)
+
+
+def cast_bool(value):
```

Comment:
MINOR: not relevant in this case but since we start adding typing it would be good to exercise it :-)
`` def cast_bool( value: str) -> bool`` . In this case is definitively useful since the cast is from string to bool ...

Reply:
took the opportunity to add typing to all function definitions